### PR TITLE
fix: replace username with full name in the learner record title

### PR DESCRIPTION
**Pull request by ahtesham-quraish**
Description: Remove username from learner record title and replace it with Full Name

[VAN-1832](https://2u-internal.atlassian.net/browse/VAN-1832)

| Before | After |
| ------------- | ------------- |
|  image| image |

### CODE REVIEW

**Review by syedsajjadkazmii — APPROVED** [review]

**Comment by arbrandes** [comment]
Why is the username being removed?  Has this gotten product review, yet?  Apologies if it already has, but I'd like this to be confirmed before we merge it.

**Comment by hurtstotouchfire** [comment]
@arbrandes I was expecting this to be swapping username for full name, which made sense to us since this is a transcript page, basically. Email should be retained because it's used by the credit transfer feature. Open to leaving username if needed, but typically full name is more appropriate for a certificate record.

**Comment by arbrandes** [comment]
@hurtstotouchfire, thanks for the description of the intended behavior.  The link the ticket description is private, unfortunately, so there was no way to understand the reasoning the change other than figuring it out from the code.

Still, it's a user-facing change, and we should probably get somebody from Product giving it a 👍🏼.  Adding the label in.

**Comment by hurtstotouchfire** [comment]
Sounds good, thanks

File: src/components/ProgramRecord/ProgramRecordHeader.jsx
Comment (hurtstotouchfire) [inline src/components/ProgramRecord/ProgramRecordHeader.jsx:84]: Maybe this is just me but I feel like we need non-breaking spaces around this pipe? Defer to UX, but it's hard to read the name in the screenshots, it just flows into the email address.

I know this was already a problem with username, but while we're here improving things...

Comment (zainab-amir) [reply src/components/ProgramRecord/ProgramRecordHeader.jsx:84]: Yes, I agree.

Comment (ahtesham-quraish) [reply src/components/ProgramRecord/ProgramRecordHeader.jsx]: Done

**Comment by zainab-amir** [comment]
> Does this change completely remove the association of usernames from learner records? If so, does that impact any future plans for reports that may touch learner records?

This is just a user facing change. The username has been replaced with full name only on the frontend. All the existing functionality that depends on username for learner record has not been touched.

**Comment by crathbun428** [comment]
@jmakowski1123 @bmtcril - Capturing this question here since I think its an important one. Curious how an instructor may handle identifying a learner that has changed their email address if they have a number of learners with the same name. Will there be a way of identifying those learners in this case?

**Comment by jmakowski1123** [comment]
Got it, thanks @zainab-amir . 

@crathbun428 Since this is just a frontend change, I'm assuming the data pipeline won't be affected since the user ids aren't being removed from the backend. Something to pass back to Brian, but also to keep in mind down the road if the conversation evolves to removing usernames entirely (which I think is a future state discovery conversation, but not being acted on at the moment).

This has product aproval!

**Review by justinhynes — APPROVED** [review]

File: src/components/ProgramRecord/ProgramRecordHeader.jsx
Comment (justinhynes) [inline src/components/ProgramRecord/ProgramRecordHeader.jsx:84]: @ahtesham-quraish -- Having the `|` character as part of the translated string seems... strange. Also, I question why this needs to be a `FormattedMessage` here when it is the name of a user, which will never be translated.

**Comment by hurtstotouchfire** [comment]
> @jmakowski1123 @bmtcril - Capturing this question Brian raised here since I think its an important one. Curious how an instructor may handle identifying a learner that has changed their email address if they have a number of learners with the same name. Will there be a way of identifying those learners in this case?

@crathbun428 This page is a learner-facing transcript. Instructors have no way of navigating to this page. This transcript is used by the learner to track their own progress and can also be shared publicly (like with a CV or sending to a university for transfer credit).

**Review by arbrandes — APPROVED** [review]
Looks good to me!

**Pull request merged**
